Core/FileLoaders/LocalFileLoader.cpp

```diff
@@ -73,23 +75,19 @@ std::string LocalFileLoader::Path() const {
 	return filename_;
 }
 
-void LocalFileLoader::Seek(s64 absolutePos) {
-#ifdef __ANDROID__
-	lseek64(fd_, absolutePos, SEEK_SET);
-#else
-	fseeko(f_, absolutePos, SEEK_SET);
-#endif
-}
-
-size_t LocalFileLoader::Read(size_t bytes, size_t count, void *data, Flags flags) {
-#ifdef __ANDROID__
-	return read(fd_, data, bytes * count) / bytes;
+size_t LocalFileLoader::ReadAt(s64 absolutePos, size_t bytes, size_t count, void *data, Flags flags) {
+#ifndef _WIN32
+	return pread(fd_, data, bytes * count, absolutePos) / bytes;
```

Comment:
bytes * count could overflow here, but given the fact that no PSP game is over 2GB anyway, I don’t see how that ever conceivably could be a call.

Reply:
Turns out, for posterity, that Android doesn't seem to like pread64 after all (#10862) - bah, silly bionic.

-[Unknown]